test/unit/test_serialize_bp.c

```diff
@@ -0,0 +1,81 @@
+// SPDX-FileCopyrightText: 2021 DMaroo <dhruvmaru007@gmail.com>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_util/rz_serialize.h>
+#include <rz_bp.h>
+#include <rz_debug.h>
+#include "minunit.h"
+
+bool test_bp_save_load() {
+	Sdb *bp_sdb = sdb_new0();
+	mu_assert_notnull(bp_sdb, "failed to create sdb");
+	RzDebug *debug_save = rz_debug_new(0);
+	mu_assert_notnull(debug_save, "failed to create debug instance");
+
+	RzBreakpointItem *item_save = rz_debug_bp_add(debug_save, 0x1337, 0, true, 0, "doom", 42);
+	mu_assert_notnull(item_save, "failed to create bp item");
```

Comment:
Tests are failing here. I think I am creating a breakpoint at a memory location non-existent in the current debug instance, however, I am not sure how to resolve this. Will require some help, @thestr4ng3r.